my-app/src/components/VideoCard/ViCard.css

```diff
@@ -46,49 +37,37 @@
 }
 
 .content{
-    margin-left: 2rem;
+    margin-left: 1rem;
     margin-top: 1rem;
     margin-bottom: 1rem;
     flex-wrap: wrap;
 }
 
-.footer{
+.card_footer{
     display: flex;
-    margin-bottom: 10px;
     justify-content: space-around;  
     border: none;
-    flex-wrap: wrap;
+    align-items: center;
 }
-.footer__buttons{
+.footer_info{
     display: flex;
     justify-content: space-around;
     margin-left:auto ;
```

Comment:
Code format and space after : is not consistent

Reply:
did the formatting